modules/integration/tests-common/admin-clients/src/main/java/org/wso2/mb/integration/common/clients/operations/utils/AndesClientUtils.java

```diff
@@ -316,24 +319,18 @@ public static void createMockFile(String filePathToRead, String filePathToCreate
             }
 
         }
+
+        //If already exists, deleting the file
+        if (deleteRandomFile(filePathToCreate)) {
+            log.info("File requested to create already exists. Deleting file... " + filePathToCreate);
```

Comment:
The deletion happens in  deleteRandomFile(..) ? if so shouldn't the info be changed to mention whether its deleted or not ? 'deleting..' sounds like its about to be deleted.